python/ql/src/experimental/semmle/python/Concepts.qll

```diff
@@ -13,3 +13,40 @@ private import semmle.python.dataflow.new.DataFlow
 private import semmle.python.dataflow.new.RemoteFlowSources
 private import semmle.python.dataflow.new.TaintTracking
 private import experimental.semmle.python.Frameworks
+private import semmle.python.ApiGraphs
+
+module LDAPQuery {
+  abstract class Range extends DataFlow::Node {
+    abstract DataFlow::Node getLDAPNode();
+
+    abstract string getLDAPPart();
```

Comment:
This naming feels ambiguous. Is LDAP Part an actual concept in LDAP?

Reply:
I agree. I'm waiting for the review to come into an agreement about how to name it :)

Reply:
Closing this since `LDAPPart` idea has been removed.